pkg/koordlet/util/system/config.go

```diff
@@ -77,30 +78,32 @@ func InitSupportConfigs() {
 
 func NewHostModeConfig() *Config {
 	return &Config{
-		CgroupRootDir:         "/sys/fs/cgroup/",
-		ProcRootDir:           "/proc/",
-		SysRootDir:            "/sys/",
-		SysFSRootDir:          "/sys/fs/",
-		VarRunRootDir:         "/var/run/",
-		VarLibKubeletRootDir:  "/var/lib/kubelet/",
-		RunRootDir:            "/run/",
-		RuntimeHooksConfigDir: "/etc/runtime/hookserver.d",
-		DefaultRuntimeType:    "containerd",
+		CgroupRootDir:                "/sys/fs/cgroup/",
+		ProcRootDir:                  "/proc/",
+		SysRootDir:                   "/sys/",
+		SysFSRootDir:                 "/sys/fs/",
+		VarRunRootDir:                "/var/run/",
+		VarLibKubeletRootDir:         "/var/lib/kubelet/",
+		RunRootDir:                   "/run/",
+		RuntimeHooksConfigDir:        "/etc/runtime/hookserver.d",
+		DefaultRuntimeType:           "containerd",
+		HAMICoreLibraryDirectoryPath: "/data/bin/libvgpu.so",
 	}
 }
 
 func NewDsModeConfig() *Config {
 	return &Config{
 		CgroupRootDir: "/host-cgroup/",
 		// some dirs are not covered by ns, or unused with `hostPID` is on
-		ProcRootDir:           "/proc/",
-		SysRootDir:            "/host-sys/",
-		SysFSRootDir:          "/host-sys-fs/",
-		VarRunRootDir:         "/host-var-run/",
-		VarLibKubeletRootDir:  "/var/lib/kubelet/",
-		RunRootDir:            "/host-run/",
-		RuntimeHooksConfigDir: "/host-etc-hookserver/",
-		DefaultRuntimeType:    "containerd",
+		ProcRootDir:                  "/proc/",
+		SysRootDir:                   "/host-sys/",
+		SysFSRootDir:                 "/host-sys-fs/",
+		VarRunRootDir:                "/host-var-run/",
+		VarLibKubeletRootDir:         "/var/lib/kubelet/",
+		RunRootDir:                   "/host-run/",
+		RuntimeHooksConfigDir:        "/host-etc-hookserver/",
+		DefaultRuntimeType:           "containerd",
+		HAMICoreLibraryDirectoryPath: "/data/bin/libvgpu.so",
```

Comment:
Why the default path is `/data/bin/libvgpu.so`? According to [hami chart](https://github.com/Project-HAMi/HAMi/blob/master/charts/hami/templates/device-plugin/daemonsetnvidia.yaml), the default host path is `/usr/local/vgpu/libvgpu.so`.

Reply:
OK, I will fix it